code/modules/client/verbs/who.dm

```diff
@@ -80,6 +80,8 @@
 		for(var/client/C in admins)
 			if(!C.holder.fakekey)
 				msg += "\t[C] is a [C.holder.rank]\n"
+			if(C.is_afk())
+				msg += " (AFK)"
```

Comment:
Also, it adds \n to the end of the line, so the ` (AFK)` bit would end up on the next line.